src/map/clif.c

```diff
@@ -9348,21 +9351,25 @@ void clif_parse_LoadEndAck(int fd, struct map_session_data *sd) {
 
 	if(sd->vd.cloth_color)
 		clif->refreshlook(&sd->bl,sd->bl.id,LOOK_CLOTHES_COLOR,sd->vd.cloth_color,SELF);
+
 	if (sd->vd.body_style)
 		clif->refreshlook(&sd->bl,sd->bl.id,LOOK_BODY2,sd->vd.body_style,SELF);
-	// item
-	clif->inventorylist(sd);  // inventory list first, otherwise deleted items in pc->checkitem show up as 'unknown item'
```

Comment:
This would be obvious because `pc_delitem` in `pc_checkitem` sends `clif_delitem`, which requires the client to have the inventory list before asking it to delete something. But okay, I'll put it back.